core/common/assets/js/api/core/data/stroages/base-storage.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
We already have a storage mechanism in our system. Consider merging it.

Reply:
I think this is best idea, but its better to add it now to TODO list, because i think will be easier for now, handle issues with that cache.

Reply:
I don't see a reason why not to use the available storage utility. If there is some missing functionality, just append it there.

Reply:
@iNewLegend  Please add a TODO.

Reply:
Handled.

Reply:
I can't agree with change